### DISCUSSION TITLE
DistributionTool v1.7 fails on SVG images in any attribute

### DISCUSSION THREAD

**Discussion by pre-martin** [elgatosf/elgato#76] [General] [answered]
When a SVG icon is referenced in the Manifest file in

1. `CategoryIcon`
2. `Actions.Icon`
3. `Actions.States.Image`

the current DistributionTool v1.7 fails with the following error messages:

1. `Error: The plugin has a category icon "actionImage.png" that does not exist.`
2. `Error: The action 'com.elgato.template.action' has an "Icon" property with the value 'action@2x.png' which is not an existing file on disk. Please add a valid value to the property "Icon" in the manifest.json`
3. `The plugin has an icon "action@2x.png" that does not exist.`

Steps to reproduce:

1. `git clone https://github.com/elgatosf/streamdeck-plugin-template.git` (or use any other Manifest with SVGs)
4. Download the DistributionTool from https://docs.elgato.com/sdk/icon-packs/packaging into the current directory
5. `cd streamdeck-plugin-template\src`
6. `..\..\DistributionTool.exe -b -i com.elgato.template.sdPlugin -o .`

Expected result:
A bundled version of the plugin.

Current result:
```
 ..\..\DistributionTool.exe -b -i com.elgato.template.sdPlugin -o .
                    DistributionTool                  using printf
                    DistributionTool                  Validating and exporting the plugin com.elgato.template.sdPlugin
                    DistributionTool                  Error: The plugin has an icon "action@2x.png" that does not exist.
                    DistributionTool                  Error: The plugin is invalid
```

**Comment by ChekTek** [comment, marked as answer]
Hi @pre-martin,

Thanks for pointing this out! The DistributionTool on the Icon Packs page was outdated. I have now replaced it with the latest version from the plugins page. Please re-download the tool and let me know if you are still having issues.

**Reply by pre-martin** [reply]
Hi. The new version works without problems. Thank you.

**Comment by erikjmurray** [comment]
Hi I am having the same issue on the Plugin distribution tool. Is there an updated version for that as well? Note: I built my own plugin. It works via the sympathetic link, but this is the first time I've tried to use the DistributionTool.

**Reply by ChekTek** [reply]
Hi @erikjmurray, yes you should download the latest version of the [DistributionTool](https://docs.elgato.com/sdk/plugins/packaging). Note if you are using rasterized images like `PNG` you will need to provide two versions of the image, i.e. `icon.png` and `icon@2x.png` and exclude the extension from the file path in the manifest.
